android/lib/pages/settings_page.dart

```diff
@@ -60,7 +60,7 @@ class _SettingsPageState extends State<SettingsPage> {
   }
 
   Scaffold settingsPage(GetSettingsOutput settings) {
-    Widget profilePicture = circleAvatarBuilder(settings.profilePictureId!, 10);
+    Widget profilePicture = circleAvatarBuilder(settings.profilePictureId, 10);
```

Comment:
Removing this null check is a correct approach. Missed this during my implementation.